CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
```

Comment:
How about "range"?  "Variety" for me puts the emphasis on the users being different from each other, and for Mypy I think what's more important is that it's good for lots of users and lots of codebases.